doc/source/docs/user_guide/mergingdata.rst

```diff
@@ -91,21 +91,21 @@ In a Spatial Join, two geometry objects are merged based on their spatial relati
 
    # Execute spatial join
 
-   cities_with_country = geopandas.sjoin(cities, countries, how="inner", predicate='intersects')
+   cities_with_country = cities.sjoin(countries, how="inner", predicate='intersects')
    cities_with_country.head()
 
 
 GeoPandas provides two spatial-join functions:
 
-- :func:`sjoin`: joins based on binary predicates (intersects, contains, etc.)
-- :func:`sjoin_nearest`: joins based on proximity, with the ability to set a maximum search radius.
+- :meth:`GeoDataFrame.sjoin`: joins based on binary predicates (intersects, contains, etc.)
+- :meth:`GeoDataFrame.sjoin_nearest`: joins based on proximity, with the ability to set a maximum search radius.
 
```

Comment:
Could we leave a short note here that both those methods are also available as top-level functions for historical reasons? (just so people only knowing the functions don't get confused about it)

Reply:
Good point. I've added notes to relevant docs pages.